FORMATTING_GUIDE.md

```diff
@@ -0,0 +1,389 @@
+
+**Example of a `copy` include:**
+
+````
+```bash
+curl -XGET "localhost:9200/_tasks?actions=*search&detailed
+```
+{% include copy.html %}
+````
+
+
+**Example of a `copy-curl` include:**
+
+````
+```json
+PUT /sample-index1/_clone/cloned-index1
+{
+  "aliases": {
+    "sample-alias1": {}
+  }
+}
+```
+{% include copy-curl.html %}
+````
+
+## Adding pages or sections
+
+When adding a page or a section, make the `nav_order` of the child pages multiples of 10. For example, if you have a parent page `Clients`, make child pages `Java`, `Python`, and `JavaScript` have a `nav_order` of 10, 20, and 30, respectively. Doing so makes inserting additional child pages easier because it does not require renumbering existing pages.
+
+## Videos
+
+To insert a video, add a youtube player include similar to the following:
+
+```
+{% include youtube-player.html id='_g46WiGPhFs' %}
+```
+
+Note that the `id` variable refers to the youtube video ID at the end of the URL. For example, the youtube video at the URL `https://youtu.be/_g46WiGPhFs` has the id `_g46WiGPhFs`. The ID must be surrounded with single quotation marks.
+
+## Math
+
+To add mathematical expressions to a page, add `has_math: true` to the page’s front matter. Then insert LaTeX math into HTML tags with the rest of your Markdown content, as shown in the following example:
+
+```
+## Math
+
+Some Markdown paragraph. Here's a formula:
+
+<p>
+  When \(a \ne 0\), there are two solutions to \(ax^2 + bx + c = 0\) and they are
+  \[x = {-b \pm \sqrt{b^2-4ac} \over 2a}.\]
+</p>
+
+And back to Markdown.
+```
+
+## Variables in curly braces
+
+To correctly display variables that are in curly braces, escape the curly braces with the `{% raw %}{% endraw %}` tags:
+
+````
+"message_template": {
+    "source": "the index is {% raw %}{{ctx.index}}{% endraw %}"
+}
+````
+
+The variable `ctx.index` is rendered in double curly braces.
+
+## Links
+
+To add a link to a document, section, or image, use the `[name](link)` syntax, for example:
+
+```
+## Looking for the Javadoc?
+
+See [opensearch.org/javadocs/](https://opensearch.org/javadocs/).
+```
+
+### Section links
+
+**Section links** are links to headings in your document. Markdown lowercases the headings for links, drops back ticks, and replaces spaces with hyphens:
+
+```
+## The `minimum_should_match` parameter
+
+For more information, see [the `minimum_should_match` parameter](#the-minimum_should_match-parameter).
+```
+
+### Internal links
+
+**Internal links** are links to another document or image within the documentation website. Because the documentation website is versioned, do not hard code the version number in the link. Use the relative path where `{{site.url}}{{site.baseurl}}` refers to the main directory instead:
```

Comment:
Should the last sentence have commas like so?: "Use the relative path, where `{{site.url}}{{site.baseurl}}` refers to the main directory, instead:"